tests/unit/package_managers/bundler/test_parser.py

```diff
@@ -180,3 +181,25 @@ def test_parse_gemlock_empty(
 
     assert f"Package {rooted_tmp_path.path.name} is bundled with version 2.5.10" in caplog.messages
     assert result == []
+
+
+@pytest.mark.parametrize(
+    "source",
+    [
+        "https://rubygems.org",
+        "https://dedicatedprivategemrepo.com",
+    ],
+)
+@mock.patch("cachi2.core.package_managers.bundler.parser.download_binary_file")
+def test_dependnecies_could_be_downloaded(
+    mock_downloader: mock.MagicMock,
+    source: str,
+) -> None:
+    base_destination = Path("/tmp/foo")
+    dependency = GemDependency(version="0.0.2", source=source, name="foo")
```

Comment:
nitpick: let's reorder the params - name, version, source :smile:

Reply:
Done.